Lab2/Lab2.ipynb

```diff
@@ -765,33 +751,30 @@
rO172uc9Msl3gCOB7w/yLEmSJEnS1KYbFvsLYHtgV2Af4JokZw6hp/JqmuGpTwU2nHDteJoht1sCjwROT3JBVV3Y7rH5uUme95yqunayF1XV/wJ7Jnku8Ebg8AGzS5IkSZImMWVxWVU7JHkM8ALgOe33Xyc5BfgSUGvzwqo6BSDJQmDbVe1JNgIOAnatqhXAN5OcBrwYWNQWkHv3+54k9+nZMuVGmnmZkiRJkqQZMO2CPlV1HnBekr8F/oxm/uVBNIvjFPBXSW6uqiVDyLITcEdVXdLTdgHwxH4+nOQMmt7Ohyb5MHBxkncDdwC3Ai+f5rOHAYe1p7cl+fGax5dm3ANpFqqSZht/NzWb+fu5nsm7u07QN383NZs9dG0+1NdqsVV1J81iPmclORx4Os1iP88GXpDkkqraeW0C9NiYpoex1400i/H0k/HpkzQ/oc/PLgYWAyRZUlUL+/mcNEr+bmq28ndTs5m/n5qt/N3UbJZkrToP+9rnsldVrayqL1TVITRzI19Cs/jPoFYAm0xo24QBFg+SJEmSJI3GGheXvarqpqr6ZFU9YwhZLgHmJtmxp2033EJEkiRJkma9gYrLtZFkbpJ5wBxgTpJ5SeZW1U3AKcBbk2yU5HHAAcAnRhxx8YjfJ/XL303NVv5uajbz91Ozlb+bms3W6vczVWu16OtaS3IMcPSE5rdU1TFJNgM+CjwFuJ5mldhPjTSgJEmSJGmNjby4lCRJkiSte0Y+LFaSJEmStO6xuJQkSZIkDczispVksySnJrkpyS+SvKDrTFKS+yQ5sf2d/F2SHyR5Wte5pF5Jdkxya5L/6DqL1CvJIUl+0v5tvzzJ47vOJCVZkOSMJL9Ocm2S45L0tfe8NExJjkyyJMltST424dqfJflpkpuTfC3J9v080+LyD44HVtLs3flC4ENJHt5tJIm5wFXAE4FNgX8EPptkQZehpAmOB87vOoTUK8lTgHcDLwPuBzwBuKLTUFLjg8AyYGvgkTR/41/ZZSCtt64G3k6zoOpdkjyQZhePfwQ2A5YA/9nPAy0ugSQbAQcB/1hVK6rqm8BpwIu7Tab1XbuX7DFV9fOqurOqvgj8DNi962wSND1DwG+Ar3YcRZroLcBbq+rb7f9//rKqftl1KAnYAfhsVd1aVdcCXwbs0NDIVdUpVfUFml06eh0IXFhVJ1fVrcAxwG5JHra6Z1pcNnYC7qiqS3raLsD/oWuWSbIlze/rhV1nkZJsArwVeG3XWaReSeYAC4HNk1yWZGk79HDDrrNJwL8AhyS5b5JtgKfRFJjSbPFwmloIaDo7gMvpozayuGxsDNw4oe1GmmE00qyQZAPgk8DHq+qnXeeRgLcBJ1bVVV0HkSbYEtgAeA7weJqhh48C3txhJmmVc2j+I/23wFKaIYdf6DKQNMFa10YWl40VwCYT2jYBftdBFukektwL+ATNvOAjO44jkeSRwJOBf+44ijSZW9rv/1pV11TVr4B/Ap7eYSZp1d/zM2nms20EPBD4I5r5wdJssda1kcVl4xJgbpIde9p2w6GHmgWSBDiR5l/iD6qq33ccSQLYB1gAXJnkWuB1wEFJvt9lKAmgqn5N0yNUXWeRJtgM2A44rqpuq6rrgZPwHz40u1xIUwsBd61P8xD6qI0sLrlrHPEpwFuTbJTkccABND1FUtc+BOwMPKOqblndzdKILKb5Q/PI9usE4HTgqd1Fku7mJOBVSbZI8kfAq4EvdhtJ67u2F/1nwBFJ5ia5P/BSeua3SaPS/g7OA+YAc5LMa7fFORXYNclB7fWjgB/2My3L4vIPXglsSLM09KeBI6rKnkt1qt1T6K9p/uP92iQr2q8XdptM67uqurmqrl31RTOE5taqWt51Nqn1Npotci4BfgL8AHhHp4mkxoHAfsBy4DLgduA1nSbS+urNNNMIFgEvao/f3P4tP4jm/zN/DewJHNLPA1PliBFJkiRJ0mDsuZQkSZIkDcziUpIkSZI0MItLSZIkSdLALC4lSZIkSQOzuJQkSZIkDcziUpIkSZI0MItLSZKGKMkxSar9ujPJr5Ocn+QdSbZai+e9Ick+w08qSdJwWVxKkjR8NwKPAR5Ls/H0KcCLgR8l2X0Nn/UGYJ+hppMkaQbM7TqAJEnroNur6ts952cm+RDwDeA/kzy0qu7oKJskSTPCnktJkkagqn5D0wv5EOApAEmOTfKjJCuSLE3yyd6hs0l+DjwAOLpnqO0+7bXXtsNtb0xyXZL/TvLHI/6xJEm6i8WlJEmj8zXgdmCv9nwL4J3A/sCrgQcDZyeZ015/Ns0Q2xNphtk+Bvh+e21b4DjgAOCvgDnAt5JsOuM/hSRJk3BYrCRJI1JVtyX5FbBle/7yVdfagvI8YCnwOOAbVfWDJLcDSycMs6WqXjPhs2cBy2iKzX+f6Z9FkqSJ7LmUJGm0ctdB8rQk5ya5kaZHc2l7aafVPiTZK8lZSa5vP3szsHE/n5UkaSZYXEqSNCJJ5tHMobwuyaOB02gKyhfTDHldNVx23mqeMx/4Ck2h+tc0PZ2Ppum5nPazkiTNFIfFSpI0OvvS/O09j2Y+5XLgeVVVAEm27/M5+wH3BQ6oqpvaz84FNht6YkmS+mTPpSRJI5Dk/sC7gcuA/wE2BH6/qrBsvXCSj67knr2RGwJ30gyHXeW5+I/GkqQO+UdIkqThm5tk1RDX+wG7A0fQ9DbuV1V3JDkLeHWS9wP/DTwWeNEkz/opsH+SLwMrgIuBs2lWhz0pyYnAw4HXAb+ZsZ9IkqTVsOdSkqTh25Rm6Ou5wMnAc4D/AB5RVd8DqKozgL8HDqKZe/lE4C8medbrgZuA04Hzgd2r6kfAy4A9gS8CLwAOptm2RJKkTuTuo3EkSZIkSVpz9lxKkiRJkgZmcSlJkiRJGpjFpSRJkiRpYBaXkiRJkqSBWVxKkiRJkgZmcSlJkiRJGpjFpSRJkiRpYBaXkiRJkqSB/X+oDZxEHZ42wQAAAABJRU5ErkJggg==\n",
```

Comment:
I'm just commenting this on a random graph but I just thought that maybe we could both add titles to our plots that tie into the prompt of the problem! Something I am now thinking about doing as well! 

Overall though, very nice report, well done!